src/SonarScanner.MSBuild.PreProcessor/Roslyn/RoslynRuleSetGenerator.cs

```diff
@@ -43,10 +43,7 @@ public RoslynRuleSetGenerator(IAnalysisPropertyProvider sonarProperties)
 
         public RuleAction ActiveRuleAction
         {
-            get
-            {
-                return activeRuleAction;
-            }
+            get => activeRuleAction;
```

Comment:
As far as I can tell the `get` is never used and can be safely removed.

We should consider replacing the property with a method (`SetActiveRuleAction`)

Reply:
I removed the property completely in the next #985 so I left the CodeSmell there intentionally.